frontend/src/components/TagEditor.tsx

```diff
@@ -110,6 +107,7 @@ const trialOptions: Option[] = [
      { value: 'dance', label: 'Dancing' },
      { value: 'gait_initiation', label: 'Gait Initiation' },
      { value: 'jump', label: 'Jumping' },
+     { value: 'drop_jump', label: 'Drop Jumping' },
```

Comment:
I would change the label to just 'Drop Jump'

Reply:
Done!